### ISSUE TITLE
Move pushRegistrationManager to ServiceWorker instance?

### ISSUE THREAD

**Issue by efullea** [w3c/push-api#2]
Issue originally posted by @johnmellor at
https://github.com/telefonicaid/WebAPISpecs/issues/30

See issue and discussion there, but please post new comments here.

**Comment by johnmellor** [comment]
See discussion [on public-webapps](http://comments.gmane.org/gmane.org.w3c.appformats/30135) and slightlyoff/ServiceWorker/issues/365.

Jake's suggestion (copied below) seems a very clean way of doing this, resolving the [earlier concerns](https://github.com/telefonicaid/WebAPISpecs/issues/30#issuecomment-41580520):

``` js
navigator.serviceWorker.ready.then(function(reg) {
  reg.push.register(...);
});
```

See also similar background sync issue: slightlyoff/BackgroundSync/issues/6

**Comment by mvano** [comment]
Just updating John's code sample to the latest syntax:

``` idl
navigator.serviceWorker.ready.then(function(serviceWorkerRegistration) {
  serviceWorkerRegistration.pushRegistrationManager.register().then(...);
});
```

**Comment by efullea** [comment]
Closed by #13
